pkg/resourcehandlers/github/github_resource_handler_test.go

```diff
@@ -116,7 +116,7 @@ func TestUrlToGitHubLocator(t *testing.T) {
 			false,
 			&Cache{
 				cache: map[string]*ResourceLocator{
-					"https://github.com/gardener/gardener/blob/master/docs/README.md": ghrl1,
+					"github.com:gardener:gardener:docs/readme.md:master": ghrl1,
```

Comment:
```suggestion
					"github.com:gardener:master:gardener:docs/readme.md": ghrl1,
```
Also expand this TC to two cases - 1) return valid RL resolving the API (using cache) and 2) return valid RL without resolving the API (not  using the cache)

Reply:
Add a TC for raw resource links too